src/javascripts/Services/columnsService.js

```diff
@@ -96,26 +63,22 @@ ngapp.service('columnsService', function(settingsService, nodeHelpers) {
         }
     };
 
-    let defaultColumnsConfig = {
-        customColumns: [],
-        activeColumns: [{
-            label: 'FormID',
-            width: '40%'
-        }, {
-            label: 'EditorID',
-            width: '25%'
-        }, {
-            label: 'Name'
-        }]
-    };
-
     this.loadColumns = function() {
-        let data = fh.loadJsonFile('columns.json') || defaultColumnsConfig;
-        service.columns = [formIDColumn, editorIDColumn, nameColumn];
-        data.customColumns.forEach((column) => service.addColumn(column));
-        data.activeColumns.forEach((column) => service.setColumnData(column));
+        let fileData = fh.loadJsonFile('columns.json');
+        columnsFactory.views.forEach(view => {
+            let data = (!fileData || !fileData[view.viewId]) 
+                ? view.defaultColumnsConfig : fileData[view.viewId];
+            service.views[view.viewId] = {};
```

Comment:
This wouldn't be necessary at all if the `columnsFactory` stored views on unique properties, as I previously suggested.

Reply:
Yeah, this line is kinda ugly. But this way, the views can be dynamic. Do we plan on adding that many more views?

Reply:
The views can still be dynamic with explicit keys in the `columnsFactory`.  Just do:

```js
this.getColumnsForView(viewName) {
  return columnsFactory[viewName].columns;
}
```